configure.ac

```diff
@@ -21,7 +21,7 @@
 #
 
 AC_PREREQ([2.69])
-AC_INIT([cc-oci-runtime], [2.1.1])
+AC_INIT([cc-oci-runtime], [2.1.2])
```

Comment:
Yes, working through a rebase now and will take the feedback (thanks!) into account.